web/docs/terminals.md

```diff
@@ -79,3 +79,28 @@ https://docs.microsoft.com/en-us/windows/terminal/customize-settings/profile-set
 }
 ```
 
+## Alacritty
+
+![image](alacritty.png){: align=right width=45% }
+
+[Alacritty](https://alacritty.org/) is a fast, cross-platform, OpenGL terminal emulator.
+You can download it from their [Github releases page](https://github.com/alacritty/alacritty/releases)
+
+Once you have downloaded it, Open `%APPDATA%\alacritty\alacritty.yml` in a text editor and add the following
+
+```yaml
+env:
+    TERM: alacritty
+    MSYSTEM: UCRT64 # The envrionment that you want, can be: MSYS, MINGW64, MINGW32, CLANG64, UCRT64
+    CHERE_INVOKING: "1"
+    MSYS2_PATH_TYPE: inherit
+wing
+working_directory: C:/projects # The directory that you want alacritty to open in
+
+shell:
+    program: D:/dev/programs/msys64/usr/bin/bash.exe # The shell that you want to use
```

Comment:
I'd use the default install path: `C:/msys64/usr/bin/bash.exe`

Reply:
Oh yeah, sorry forgot that, It was my personal config:)